### ISSUE TITLE
type the commands, it goes through and no gui pops up

### ISSUE THREAD

**Issue by mrdexterity** [thejoabo/autofishbot#130]
![image](https://github.com/thejoabo/autofishbot/assets/158469870/25f4ba88-6942-48e8-9384-35d94cb3e8a6)

(not using git) i download the repository, run the command, make the config with my token, channel id, api key etc. it says the command goes through and nothing happens. earlier i was having issues with using git with that command it says
![image](https://github.com/thejoabo/autofishbot/assets/158469870/e55cec41-703c-47aa-b87e-b3950a5ac223)
im not very experienced with this stuff but ive google everywhere and cant find a solution.

**Comment by dawson621** [comment]
I think you have the same issue as me, I was able to fix it hope this helps [click here](https://github.com/thejoabo/autofishbot/issues/125) also here is the version of Python I had to download [https://www.python.org/downloads/release/python-3116/](https://www.python.org/downloads/release/python-3116/) 

Reminder DON'T RUN BOT for LONG periods! Although there is no way for the dev to detect this directly he added a thing that will give you a 14-day ban for "captcha automation" if you are fishing for a long period. For me, I had the fishing bot bypassed about 3 captchas and was on about the 2-hour mark I think. (although I did skip the scheduled breaks the auto-fishing bot had). Once you trigger the "captcha automation" ban a 30-minute timer starts before it will show you the message and ban you so it will look like it came out of now where if you stopped the bot during the 30 minutes and an hour later like me went to manual fish a bit and instantly got the banned message. This type of ban does seem to get given out falsely the most from what I have seen in the virtual fisher discord too as if hit x amount of fishing trips/hit x amount of time or whatever he uses for triggeing this ban then it happens. One of the mods on the V Fishing Discord server has said that taking frequent breaks when playing can prevent it. Also, someone with a lot of knowledge on the v fishing bot (not a mod in the v fishing server) who I won't name, said that changing servers at the end of the 20-minute treasure/fish boost is a good way to prevent a ban. Attached are some screenshots below from the dev of the bot to give you some insight. 

TL;DR Take breaks and switch between servers to prevent a ban. As long as you are not fishing for about 1-2 hours straight with no breaks you should be fine and DON'T Skip the Secdualed Breaks the auto fish bot has built-in like I did.
![image](https://github.com/thejoabo/autofishbot/assets/122133472/c61413bb-f3a5-4d26-9eef-8a87cf5a1ee0) 
![image](https://github.com/thejoabo/autofishbot/assets/122133472/7ea00d71-3f21-4116-9203-fa4b02152c8a)
*all of this is just my understanding of it and have not tested any of the tips I have found.
